### PR TITLE
React v0.14.0-rc1 and React-Router v1.0.0-rc1 support.

### LINKED ISSUE #70: React 0.14 support
Any plans on supporting React v0.14? (and react-router 1.0?)

Currently getting a lot of these
```Warning: require('react/addons') is deprecated. Access using require('react-addons-{addon}') instead.```

Should we open up a react 0.14 branch?

### PR DESCRIPTION
**Pull request by NogsMPLS**
Not sure how to create a branch on elemental repo itself, but here is a PR that gets rid of all react 0.14 and react-router 1.0 release candidate warnings. (with the one exception of ReactCSSTransitionGroup that has to deal with dropdown animation that I don't have time to get into right now).

Possible to get this merged into a react 14 branch in prep for when 14 actually goes live?

Closes #70

### CODE REVIEW

File: site/src/site.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-	{ value: 'modal',       label: 'Modal' },
-	{ value: 'misc',        label: 'Misc' }
+	{ value: '/css',         label: 'CSS' },
+	{ value: '/grid',        label: 'Grid' },
+	{ value: '/buttons',     label: 'Buttons' },
+	{ value: '/glyphs',      label: 'Glyphs' },
+	{ value: '/forms',       label: 'Forms' },
+	{ value: '/spinner',     label: 'Spinner' },
+	{ value: '/modal',       label: 'Modal' },
+	{ value: '/misc',        label: 'Misc' }
```
Comment (NogsMPLS) [inline site/src/site.js]: new react router gets rid of names, only way I could get stuff working was changing this value to an actual path.

**Comment by zackify** [comment]
@NogsMPLS please change the base to the react 14 branch I just created and I'll merge this!

**Comment by NogsMPLS** [comment]
okay, let me close and re-pull request

**Comment by Guria** [comment]
There is no need to open new issue in such cases. Github will update pr if new commits added or force push to the same branch.

**Pull request closed without merging**